iOS/DuckDuckGo/CreditCardManagement/List/AutofillCreditCardListView.swift

```diff
@@ -81,38 +103,6 @@ private struct EmptyStateView: View {
     }
 }
 
-private struct CreditCardRow: View {
```

Comment:
I moved this (and it’s view model) out into into their own files as I’m re-using it in the save & fill prompts